avm/res/sql/managed-instance/main.bicep

```
@@ -57,7 +57,7 @@ param servicePrincipal string = 'None'
 param managedInstanceCreateMode string = 'Default'
 
 @description('Optional. The resource ID of another managed instance whose DNS zone this managed instance will share after creation.')
-param dnsZonePartner string = ''
+param dnsZonePartner string?
```

```suggestion
param dnsZonePartnerResourceId string?
```
... I guess, as per the description

---

Updated